laterpay/application/Controller/Admin/Pricing.php

```diff
@@ -394,34 +463,44 @@ protected function set_category_default_price( LaterPay_Core_Event $event ) {
             return;
         }
 
-        $category_id                = $term->term_id;
-        $category_price_model       = LaterPay_Model_CategoryPriceWP::get_instance();
-        $category_price_id          = $category_price_model->get_price_id_by_category_id( $post_category_id );
-        $delocalized_category_price = $price_category_form->get_field_value( 'price' );
+        $cat_ids = array_column( $categories, 'term_id' );
+        $cat_names = array_column( $categories, 'name' );
 
-        if ( empty( $category_id ) ) {
-            $event->set_result(
-                array(
-                    'success' => false,
-                    'message' => __( 'There is no such category on this website.', 'laterpay' ),
-                )
-            );
-            return;
+        asort($cat_ids);
```

Comment:
on a 2nd thought do we need to sort it if we are storing same way in other term meta as well? 

Problem with sorting is client will lose their order of term they entered.

also, we need to handle deleted terms gracefully as other term meta will have a reference to it, Is it being taken care of as well while displaying?